cloud/docker/docker.py

```diff
@@ -772,6 +778,7 @@ def get_start_params(self):
             'privileged': self.module.params.get('privileged'),
             'links': self.links,
             'network_mode': self.module.params.get('net'),
+            'read_only': self.module.params.get('read_only'),
```

Comment:
Ah, docker-py has supported read_only since https://github.com/docker/docker-py/commit/e379e8ae241faf0d8fc244cf4f7228e615149c8b, which looks like it was released in 1.0.0.